static/js/publisher/pages/Releases/components/releasesTable/cellViews.js

```diff
@@ -73,7 +73,7 @@ const ProgressiveTooltip = ({ revision, previousRevision }) => {
       <strong>{previousRevision?.revision || "Unknown"}</strong>
       <br />
       <strong>
-        {Math.round(100 - revision?.progressive["current-percentage"])}% →{" "}
+        {Math.round(100 - revision?.progressive?.["current-percentage"])}% →{" "}
```

Comment:
This leads to NaN if `revision?.progressive?.["current-percentage"]` returns undefined, maybe add a `|| 0` like below?
```suggestion
        {Math.round(100 - revision?.progressive?.["current-percentage"]) || 0}% →{" "}
```